sabnzbd/skintext.py

```diff
@@ -428,6 +428,14 @@
     "explain-complete_dir": TT(
         "Location to store finished, fully processed downloads.<br /><i>Can be overruled by user-defined categories.</i>"
     ),
+    "opt-complete_free": TT("Minimum Free Space for Completed Download Folder"),
+    "explain-complete_free": TT(
+        'Auto-pause when free space is beneath this value. Only valid for the default disk. <br /><i>In bytes, optionally follow with K,M,G,T. For example: "800M" or "8G"</i>'
+    ),
+    "opt-fulldisk_autoresume": TT("Automatic resume"),
+    "explain-fulldisk_autoresume": TT(
+        "If SABnzbd is paused because of low disk space it will check regularly and resume downloading if enough space has been freed. Needs restart if currently paused."
```

Comment:
One more question: `Needs restart if currently paused` ? Why is that?

Reply:
It's a bad way to write that if it is currently paused because the disk was full, then the automatic resume will not be applied to the current pause.  It needs to be started again manually first. I have tried rewriting it.